code/modules/shuttle/arrivals.dm

```diff
@@ -7,6 +7,7 @@
 	height = 15
 	dir = WEST
 	port_angle = 180
+	preferred_direction = WEST

```

Comment:
The docking ports originally on the maps were way out of date and didn't work. preferred_direction exists because there's usually two stationary ports for things that may be angled differently (e.g. Centcomm/Escape on Metastation). Since the arrivals shuttle has no second port it can always face the proper direction.